JUnit version 4.12
....E...E.......E.....E..E..E.E......E.....E....E.....E
Time: 2.058
There were 11 failures:
1) test24(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.ScriptableObject.buildClassCtor(ScriptableObject.java:1261)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.buildValue0(LazilyLoadedCtor.java:138)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.access$000(LazilyLoadedCtor.java:51)
	at com.google.javascript.rhino.head.LazilyLoadedCtor$1.run(LazilyLoadedCtor.java:123)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.buildValue(LazilyLoadedCtor.java:119)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.init(LazilyLoadedCtor.java:99)
	at com.google.javascript.rhino.head.ScriptableObject$GetterSlot.getValue(ScriptableObject.java:345)
	at com.google.javascript.rhino.head.ScriptableObject.get(ScriptableObject.java:494)
	at com.google.javascript.rhino.head.IdScriptableObject.get(IdScriptableObject.java:361)
	at com.google.javascript.rhino.head.ImporterTopLevel.get(ImporterTopLevel.java:129)
	at com.google.javascript.rhino.head.ScriptableObject.getProperty(ScriptableObject.java:2213)
	at com.google.javascript.rhino.head.TopLevel.cacheBuiltins(TopLevel.java:114)
	at com.google.javascript.rhino.head.ScriptRuntime.initStandardObjects(ScriptRuntime.java:262)
	at com.google.javascript.rhino.head.Context.initStandardObjects(Context.java:1074)
	at com.google.javascript.rhino.head.ImporterTopLevel.initStandardObjects(ImporterTopLevel.java:106)
	at com.google.javascript.rhino.head.ImporterTopLevel.<init>(ImporterTopLevel.java:87)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test24(NativeArray_ESTest.java:301)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
2) test04(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.AssertionError: Exception was not thrown in com.google.javascript.rhino.head.DefaultErrorReporter but in com.google.javascript.rhino.head.Context.reportRuntimeError(Context.java:947): com.google.javascript.rhino.head.EvaluatorException: Property 10000 not found.
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:112)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:49)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test04(NativeArray_ESTest.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
3) test43(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test43(NativeArray_ESTest.java:494)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
4) test06(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.AssertionError: Exception was not thrown in com.google.javascript.rhino.head.DefaultErrorReporter but in com.google.javascript.rhino.head.Context.reportRuntimeError(Context.java:947): com.google.javascript.rhino.head.EvaluatorException: Property -1911460416 not found.
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:112)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:49)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test06(NativeArray_ESTest.java:111)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
5) test07(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.AssertionError: Exception was not thrown in com.google.javascript.rhino.head.DefaultErrorReporter but in com.google.javascript.rhino.head.Context.reportRuntimeError(Context.java:947): com.google.javascript.rhino.head.EvaluatorException: Invalid JavaScript value of type class com.google.javascript.rhino.head.NativeObject
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:112)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:49)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test07(NativeArray_ESTest.java:128)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
6) test08(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.AssertionError: Exception was not thrown in com.google.javascript.rhino.head.DefaultErrorReporter but in com.google.javascript.rhino.head.Context.reportRuntimeError(Context.java:947): com.google.javascript.rhino.head.EvaluatorException: Property 114 not found.
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:112)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:49)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test08(NativeArray_ESTest.java:144)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
7) test09(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.ScriptableObject.buildClassCtor(ScriptableObject.java:1261)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.buildValue0(LazilyLoadedCtor.java:138)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.access$000(LazilyLoadedCtor.java:51)
	at com.google.javascript.rhino.head.LazilyLoadedCtor$1.run(LazilyLoadedCtor.java:123)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.buildValue(LazilyLoadedCtor.java:119)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.init(LazilyLoadedCtor.java:99)
	at com.google.javascript.rhino.head.ScriptableObject$GetterSlot.getValue(ScriptableObject.java:345)
	at com.google.javascript.rhino.head.ScriptableObject.get(ScriptableObject.java:494)
	at com.google.javascript.rhino.head.IdScriptableObject.get(IdScriptableObject.java:361)
	at com.google.javascript.rhino.head.ImporterTopLevel.get(ImporterTopLevel.java:129)
	at com.google.javascript.rhino.head.ScriptableObject.getProperty(ScriptableObject.java:2213)
	at com.google.javascript.rhino.head.TopLevel.cacheBuiltins(TopLevel.java:114)
	at com.google.javascript.rhino.head.ScriptRuntime.initStandardObjects(ScriptRuntime.java:262)
	at com.google.javascript.rhino.head.Context.initStandardObjects(Context.java:1074)
	at com.google.javascript.rhino.head.ImporterTopLevel.initStandardObjects(ImporterTopLevel.java:106)
	at com.google.javascript.rhino.head.ImporterTopLevel.<init>(ImporterTopLevel.java:87)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test09(NativeArray_ESTest.java:151)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
8) test15(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test15(NativeArray_ESTest.java:215)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
9) test10(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test10(NativeArray_ESTest.java:161)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
10) test12(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.AssertionError: Exception was not thrown in com.google.javascript.rhino.head.DefaultErrorReporter but in com.google.javascript.rhino.head.Context.reportRuntimeError(Context.java:947): com.google.javascript.rhino.head.EvaluatorException: Property 48 not found.
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:112)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:49)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test12(NativeArray_ESTest.java:187)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
11) test19(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.NativeArray.getDefaultValue(NativeArray.java:547)
	at com.google.javascript.rhino.head.ScriptRuntime.toNumber(ScriptRuntime.java:410)
	at com.google.javascript.rhino.head.ScriptRuntime.toUint32(ScriptRuntime.java:1205)
	at com.google.javascript.rhino.head.NativeArray.getLengthProperty(NativeArray.java:737)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test19(NativeArray_ESTest.java:253)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

FAILURES!!!
Tests run: 44,  Failures: 11

